app/images.py

```diff
@@ -26,7 +31,83 @@ def run(self):
             time.sleep(self.interval)
 
             print('Image Pull Complete!')
-  
+
+def resize_image(filename):
+    img = Image.open(filename)
+
+    file_type = os.path.splitext(filename)[-1]
+    out_file = 'static/img/resized_image%s' % file_type
+
+    # Check to see which is bigger (width or height)
+    
+    if img.size[0] >= img.size[1] : # width > height
+        basewidth = config['screen_size']['width']
+        
+        wpercent = (basewidth / float(img.size[0]))
+        hsize = int((float(img.size[1]) * float(wpercent)))
+        img = img.resize((basewidth, hsize), PIL.Image.ANTIALIAS)
+        img.save(out_file)
+    else:
+        baseheight = config['screen_size']['height']
+        
+        hpercent = (baseheight / float(img.size[1]))
+        wsize = int((float(hpercent) * img.size[0]))
+        img = img.resize((wsize, baseheight), PIL.Image.ANTIALIAS)
+        img.save(out_file)
```

Comment:
There is repeated code here and the scaling factor doesn't take into account when the width is largest (landscape images) but the height needs to be scaled more because of the screen's aspect ration, e.g.:
```python
scale = [
    float(img.size[0]) / float(config['screen_size']['width']),
    float(img.size[1]) / float(config['screen_size']['height'])
]
if scale[0] >= scale[1]:
    percent = 1.0 / scale[0]
else:
    percent = 1.0 / scale[1]
new_size = (
    int(float(img.size[0]) * percent),
    int(float(img.size[1]) * percent)
)
img = img.resize(new_size, PIL.Image.ANTIALIAS)
img.save(out_file)
```